src/gui/transferlistsortmodel.cpp

```diff
@@ -40,16 +40,26 @@
 namespace
 {
     template <typename T>
-    bool customLessThan(const T left, const T right)
+    int threeWayCompare(const T &left, const T &right)
+    {
+        if (left == right)
+            return 0;
+        return (left < right) ? -1 : 1;
+    }
+
+    template <typename T>
+    int customCompare(const T left, const T right)
```

Comment:
Why parameters aren't passed by references here (unlike threeWayCompare)?

Reply:
It only deals with arithmetic types